pkg/controller/common.go

```diff
@@ -1439,17 +1440,20 @@ func (vq *VolumeReplicaRequestsPriorityQueue) DrainQueue() []*ReplicaRequest {
 
 ///Removes replica requests from the priority queue and adds to operation queue.
 func (c *SharedState) tryCreateFailedReplicas(ctx context.Context, requestor operationRequester, azr azReconciler) error {
```

Comment:
This function doesn't ever return an error, so we can get rid of the return value here.